guidelines/index.html

```diff
@@ -1878,48 +1877,48 @@ <h1 id="title" class="title">Web Content Accessibility Guidelines (WCAG) 2.2</h1
    
     
     
     <div class="doclinks"><a href="https://www.w3.org/WAI/WCAG22/Understanding/accessible-authentication-minimum.html">Understanding Accessible Authentication (Minimum)</a> <span class="screenreader">|</span> <br><a href="https://www.w3.org/WAI/WCAG22/quickref/#accessible-authentication-minimum">How to Meet Accessible Authentication (Minimum)</a></div><p class="conformance-level">(レベル AA)</p>
 	<p class="change">[New]</p>
     
-    <p>A <a href="#dfn-cognitive-function-test" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-cognitive-function-test-1" title="[New]">cognitive function test</a> (such as remembering a password or solving a puzzle) is not required for any step in an authentication <a href="#dfn-processes" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-processes-3" title="ある活動を完了させるために必要な利用者の一連の動作。">process</a> unless that step provides at least one of the following:</p> 
+    <p><a href="#dfn-cognitive-function-test" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-cognitive-function-test-1" title="[New]">認知機能テスト</a> (パスワードを覚える、パズルを解く、など) は、認証<a href="#dfn-processes" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-processes-3" title="ある活動を完了させるために必要な利用者の一連の動作。">プロセス</a>のどのステップにおいても要求されない。ただし、そのステップが次の少なくとも一つを提供する場合は除く。</p> 
     <dl>
-        <dt>Alternative</dt>
-        <dd>Another authentication method that does not rely on a cognitive function test.</dd>
-        <dt>Mechanism</dt>
-      <dd>A <a href="#dfn-mechanism" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-mechanism-13" title="結果を得るためのプロセス又は手法。">mechanism</a> is available to assist the user in completing the cognitive function test.</dd>
-        <dt>Object Recognition</dt>
-        <dd>The cognitive function test is to recognize objects.</dd>
-        <dt>Personal Content</dt>
-      <dd>The cognitive function test is to identify <a href="#dfn-non-text-content" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-non-text-content-2" title="プログラムによる解釈が可能な文字の並びではないコンテンツ、又は文字の並びが自然言語においても何をも表現していないコンテンツ。">non-text content</a> the user provided to the Web site.</dd>
+        <dt>代替手段</dt>
+        <dd>認知機能テストに依存しない別の認証方法がある。</dd>
+        <dt>メカニズム</dt>
+        <dd>利用者が認知機能テストの完遂するを支援する<a href="#dfn-mechanism" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-mechanism-13" title="結果を得るためのプロセス又は手法。">メカニズム</a>が利用できる。</dd>
+        <dt>オブジェクトの認識</dt>
+        <dd>認知機能テストは、オブジェクトを認識させるものである。</dd>
+        <dt>個人的なコンテンツ</dt>
+        <dd>認知機能テストは、利用者がウェブサイトに提供した<a href="#dfn-non-text-content" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-non-text-content-2" title="プログラムによる解釈が可能な文字の並びではないコンテンツ、又は文字の並びが自然言語においても何をも表現していないコンテンツ。">非テキストコンテンツ</a>を識別させるものである。</dd>
```

Comment:
the user →「利用者」としていますが、ここの利用者は認証を利用しようとしている利用者本人であるということが重要です (他のユーザーがアップロードした画像ではダメ)。 the を明示的に訳出したほうが良いように思いますがどうでしょうか。
「その利用者」か、「その利用者本人」くらいに補っても良いように思いました。

```suggestion
        <dd>認知機能テストは、その利用者本人がウェブサイトに提供した<a href="#dfn-non-text-content" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-non-text-content-2" title="プログラムによる解釈が可能な文字の並びではないコンテンツ、又は文字の並びが自然言語においても何をも表現していないコンテンツ。">非テキストコンテンツ</a>を識別させるものである。</dd>
```

Reply:
御意です！